common/game_info.c

```diff
@@ -83,23 +84,23 @@ void    active_player(t_player_info *player)
   switch (player->num_player)
     {
     case 0:
-      player->x = 72;
-      player->y = 72;
+      player->x = (I_BEGIN + 1) * 48; // = 200px
```

Comment:
I really think we should change I_BEGIN and J_BEGIN by variables containing reference to height and width

Reply:
Some guys would say that we could set the x and y by using a variable to change the + part